code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -2,10 +2,6 @@
 	mob_type_allowed_typecache = list(/mob/living/silicon)

```

Comment:
Check the line above if you're reviewing.

Reply:
I did actually remove that line, but git being git determined the diff was below.